Source/WebCore/PlatformWPE.cmake

```diff
@@ -5,6 +5,10 @@ if (USE_CAIRO)
     include(platform/FreeType.cmake)
 elseif (USE_SKIA)
     include(platform/Skia.cmake)
+
+    list(APPEND WebCore_SOURCES
+        page/skia/MemoryReleaseSkia.cpp
```

Comment:
Why isn't this in `platform/Skia.cmake`?

Reply:
Correction this should be in `Source/WebCore/platform/SourcesSkia.txt`.

Reply:
I thought it wouldn't be right to include paths from different part of the tree, but you're right, I can see e.g. `SourcesLibWebRTC.txt` is doing just that, so it looks to be a common approach

Reply:
It's not a platform file, but a WebCore/page file

Reply:
done